src/components/Lineage/Graph/index.tsx

```diff
@@ -67,18 +70,29 @@ export function EgeriaLineageGraph(props: Props) {
   const onTabChange = async (value: any) => {
     setLoading(true);
 
-    await fetchData(uri(value));
+    await fetchData(uri(value,'false'));
 
     // TODO: extract URL to URL Map
     const path = `/asset-lineage/${guid}/${value}`;
 
     await navigateTo(path);
   };
 
+  const onSwitchChange = async (value: any, includeProcess:any) => {
+    setLoading(true);
+
+    await fetchData(uri(value, includeProcess));
+
+    // TODO: extract URL to URL Map
+    const path = `/asset-lineage/${guid}/${value}?includeProcess=${includeProcess}`;
+
+    await navigateTo(path);
```

Comment:
question: navigating to the same, but new, page wouldn't trigger the same initial `fetchData` call? I'm trying to understand why another API call is added here.